lib/plugins/aws/configCredentials/awsConfigCredentials.test.js

```diff
@@ -138,6 +139,18 @@ describe('AwsConfigCredentials', () => {
       });
     });
 
+    it('should create the .aws/credentials file if not yet present', () => {
```

Comment:
test should be independent. you cannot assume that something was created in previous test case (imagine concurrent tests run). Each test case should end with exactly the same state of the system/env as it was before test start, so if test creates some folder it should be removed after test.